website/blog/2025-01-20-podman-quadlet.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Podman Quadlets with Podman Desktop
+description: Learn how to create & manage and use Quadlets with Podman Desktop
+slug: podman-quadlet
+authors: [axel7083]
+tags: [podman-desktop, extension, podman, quadlet, systemd]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.15/banner.png
+---
+
+import ReactPlayer from 'react-player'
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+![banner](img/ai-lab-first-app/banner.png)
+
+Containers are typically deployed in Kubernetes clusters.
+However, for smaller-scale use cases such as on a single-node server or during development, Kubernetes can be overkill.
+
+What’s a more lightweight solution for running autonomous applications with multiple interacting containers?
+
+In this blog, we'll dive into what Quadlets are, their benefits, and how to utilize them within Podman Desktop.
+
+<!-- truncate -->
+
+## What Are Podman Quadlets?
+
+Podman Quadlets allow you to manage containers declaratively using systemd.
```

Comment:
Maybe point to https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html for SEO.